boot/menhir/menhirLib.ml

```diff
@@ -3792,5 +3745,5 @@ module MakeEngineTable (T : TableFormat.TABLES) = struct
 end
 end
 module StaticVersion = struct
-let require_20210419 = ()
+let require_20220210 = ()
```

Comment:
If we are increasing the menhir version, shouldn't we move to the last released version?

Reply:
In my book, any move to a more recent version is fine.